packages/clerk-js/src/ui/ImpersonationFab/ImpersonationFab.tsx

```diff
@@ -80,7 +84,7 @@ const FabContent = ({ title, signOutText }: FabContentProps) => {
         })}
         localizationKey={signOutText}
         onClick={async () => {
-          await signOut({ sessionId: session.id });
```

Comment:
❓ @desiprisg Is this branch fully rebased? We need to preserve the "logout only from current session" requirement

Reply:
Will do